src/main/java/com/conveyal/datatools/manager/jobs/RecreateBuildImageJob.java

```diff
@@ -153,7 +166,12 @@ private void terminateInstanceAndFailWithMessage(String message) {
      * Terminates the graph building instance and fails with the given message and Exception.
      */
     private void terminateInstanceAndFailWithMessage(String message, Exception e) {
-        ServerController.terminateInstances(ec2, graphBuildingInstances);
+        try {
+            ServerController.terminateInstances(parentDeployJob.getEC2ClientForDeployJob(), graphBuildingInstances);
+        } catch (Exception e2) {
```

Comment:
Exception variable names could be more descriptive since we're juggling more than one.